components/notifier/src/strategies/notification_strategy.py

```diff
@@ -0,0 +1,67 @@
+"""Strategies for notifying users about metrics."""
+
+from datetime import datetime, timedelta
+from typing import List, Optional
+from models.notification import Notification
+from models.metric_notification_data import MetricNotificationData
+
+
+class NotableEvents:
+    """Handle notification contents and status."""
+
+    def __init__(self, data_model):
+        """Initialise already notified list and store data model."""
+        self.already_notified = []
+        self.data_model = data_model
+
+    def get_notable_metrics_from_json(self, json, most_recent_measurement_seen: str) -> List[Notification]:
```

Comment:
As mentioned where this method is called: the method name says it returns notable metrics, but it really returns instances of the Notification class so a better name would be get_notifications.